### PR TITLE
ci: upload coverage to codecov

### PR DESCRIPTION
**Pull request by hugocaillard**
### Description

Upload coverage to codecov as described here https://nexte.st/book/test-coverage.html

### Applicable issues
- fixes #2932 

### Checklist
- N/A

### CODE REVIEW

File: .github/workflows/stacks-blockchain-tests.yml [hunk 1]
```diff
@@ -53,48 +53,23 @@ jobs:
...
     steps:
-      - name: Add code coverage tools
-        run: |
-          rustup component add llvm-tools-preview
-          cargo install grcov
-      - name: Add nextest
-        run: |
-          cargo install cargo-nextest
+      - name: Add nextest and cargo-llvm-cov
+        uses: taiki-e/install-action@v2
```
Comment (jcnelson) [inline .github/workflows/stacks-blockchain-tests.yml:57]: What's this?

**Comment by hugocaillard** [comment]
I'll make ot a draft PR. 
Not ready for review yet

File: .github/workflows/stacks-blockchain-tests.yml (hunk 1 above)
Comment (hugocaillard) [reply .github/workflows/stacks-blockchain-tests.yml:57]: An action to easily and quickly install rust tools. I tested it on clarinet and it's quite fast

**Comment by hugocaillard** [comment]
Closing this PR, duplicate of #4079

**Pull request closed without merging**